### PR TITLE
MultiLayerNetwork  - add getTrainingListeners

### PR DESCRIPTION
**Pull request by clavvis**
## What changes were proposed in this pull request?

In MultiLayerNetwork I created getTrainingListeners. It is preparation for solving problem: For pretraing could be necessary other listeners than for backpropagation.

## How was this patch tested?

I checked manually code.

### CODE REVIEW

File: deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/multilayer/MultiLayerNetwork.java [hunk 1]
```diff
@@ -1563,12 +1563,20 @@ protected void truncatedBPTTGradient() {
...
     public Collection<IterationListener> getListeners() {
         return listeners;
     }
 
+    /**
+    *
+    * @return trainingListeners
+    */
+   public Collection<trainingListeners> getTrainingListeners() {
+       return trainingListeners;
```
Comment (agibsonccc) [inline deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/multilayer/MultiLayerNetwork.java]: This doesn't look right. Shouldn't this be camel case? We don't have all lower case classes.

Comment (clavvis) [reply deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/multilayer/MultiLayerNetwork.java]: I don't see. First word (get) should have small letter. Next word (Training) should have capital letter to separate sense. Last word (Listeners) should have capital letter to separate sense and it's first capital letter correspond with collection name. For me it is standard name.
In MultiLayerNetwork I see a lot of similar.

Comment (AlexDBlack) [reply deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/multilayer/MultiLayerNetwork.java]: The method name is fine, but ```Collection<trainingListeners>``` won't compile. It should be ```Collection<TrainingListener>``` instead.

**Comment by clavvis** [comment]
My mistake. I have repaired it.

**Pull request merged**